packages/site/cms-site/vite.config.mts

```diff
@@ -0,0 +1,39 @@
+import react from "@vitejs/plugin-react";
+import preserveDirectives from "rollup-plugin-preserve-directives";
+import { defineConfig } from "vite";
+import dts from "vite-plugin-dts";
+import tsconfigPaths from "vite-tsconfig-paths";
+
+export default defineConfig({
+    plugins: [
+        tsconfigPaths({
+            projects: ["./tsconfig.build.json"],
+        }),
```

Comment:
Why do we need this? It seems to me that this is used for paths (e.g., `@src/`), but we're not using this in the library

Reply:
This is the only way you can give vite a custom tsconfig. The tsconfig.build.json excludes the test files: https://github.com/vivid-planet/comet/blob/main/packages/site/cms-site/tsconfig.build.json#L3 

Otherwise the test files are included in the built lib

Reply:
Ok nvm, just tested this again. The dts plugin is sufficient. I'm unsure why this didn't work the first time. I removed the paths plugin: 0845d35f2a1396795f91ec2bd875771d7eda0342